<launch>
  <!-- Start a serial interface with the vsc on "safety_port" -->
  <arg name="safety_port" default="/dev/ttyACM0"/>
  <arg name="vsc_interface_enable"  default="true"/>
    <group if="$(arg vsc_interface_enable)">
      <!-- Start a serial interface to vsc on 'safety_port' -->
        <node name="vsc_serial_interface"
              pkg="hri_safe_remote_control_system"
              type="safe_remote_control"
              respawn="true"
              respawn_delay="1">
          <param name="port" value="$(arg safety_port)"/>
          <param name="useArrowsAsAxes" type="bool" value="false"/>
        </node>
    </group>
</launch>
